manual/manual/tutorials/advexamples.etex

```diff
@@ -17,8 +16,6 @@ programming pattern known as {\em virtual types} through the example
 of window managers.
 
 \section{Extended example: bank accounts}
-\pdfsection{Extended example: bank accounts}
-\label{ss:bank-accounts}
```

Comment:
I think this label should not be deleted: it is referenced in section `3.4` .

Reply:
In fact, my intuition would be that having the labels cannot hurt, so we should/could also keep the other labels below (`ss:modules-as-classes`, etc.).